Okta.Core/OktaHttpClient.cs

```diff
@@ -175,6 +176,24 @@ public override Task<HttpResponseMessage> ExecuteAsync(HttpRequestType requestTy
                     {
                         httpClient.DefaultRequestHeaders.Remove("Authorization");
                     }
+                    if (bAddXForwardedHeader)
+                    {
+                        string ipAddress = null;
+                        if (!string.IsNullOrEmpty(content))
+                        {
+                            content = content.Trim();
+                            if (content.StartsWith("{") && content.EndsWith("}"))
+                            {
+                                var obj = JObject.Parse(content);
+                                ipAddress = (string)obj["context"]["ipAddress"];

```

Comment:
Agreed. We shouldn't be deserializing the content here.

**Quick Fix**: At the very least, we should be able to `try catch` the `JObject.Parse` method, instead of checking for `{` and `}`.